### PR TITLE
chore(server): Updated to server to freeCodeCamp

### PR DESCRIPTION
**Pull request by no-stack-dub-sack**
FCC camelCasing in `FreeCodeCamp/server/`

#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX

#### Description

Partial completion of issue #12472 - for replacing all instances of "Free Code Camp", "freecodecamp", etc. with "freeCodeCamp" within the codebase. 

This PR targets all instances within the `FreeCodeCamp/server/` directory. There's also replacements in `seed/challenges/04-video-challenges/jslingo.json`, but from here on out will be targeting one directory per PR. This way QA will hopefully not be too much of a nightmare for this if too much at once is changed. 

I tried to update just about anything a user can see and anything else that might be searchable including, email domains (from `...@freecodecamp.com` to `...@freeCodeCamp.com`), user facing URLs/links, copy, etc., while leaving alone `href` attributes and instances of `freecodecamp` in file names, etc. Hopefully this is what was intended. If anything looks wrong, please let me know and I will fix and make a new PR.

### CODE REVIEW

File: server/boot/a-extendUser.js [hunk 1]
```diff
@@ -74,8 +74,8 @@ module.exports = function(app) {
     var mailOptions = {
       type: 'email',
       to: user.email,
-      from: 'Team@freecodecamp.com',
-      subject: 'Welcome to Free Code Camp!',
+      from: 'Team@freeCodeCamp.com',
```
Comment (raisedadead) [inline server/boot/a-extendUser.js]: Revert this, and else where.

File: server/boot/a-extendUserIdent.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ export default function({ models }) {
 Your GitHub is already associated with another account.
 You may have accidentally created a duplicate account.
 No worries, though. We can fix this real quick.
-Please email us with your GitHub username: team@freecodecamp.com.
+Please email us with your GitHub username: team@freeCodeCamp.com.
```
Comment (raisedadead) [inline server/boot/a-extendUserIdent.js]: Revert this, and else where.

File: server/boot/certificate.js [hunk 3]
```diff
@@ -111,8 +111,8 @@ function sendCertifiedEmail(
 
   const notifyTeam = {
     type: 'email',
-    to: 'Michael@FreeCodeCamp.com',
-    from: 'Team@FreeCodeCamp.com',
+    to: 'Michael@freeCodeCamp.com',
```
Comment (raisedadead) [inline server/boot/certificate.js]: Revert this, and else where.

File: server/boot/certificate.js [hunk 4]
```diff
@@ -111,8 +111,8 @@ function sendCertifiedEmail(
...
+    from: 'Team@freeCodeCamp.com',
```
Comment (raisedadead) [inline server/boot/certificate.js]: Revert this, and else where.

File: server/boot/certificate.js [hunk 5]
```diff
@@ -125,7 +125,7 @@ function sendCertifiedEmail(
   const notifyUser = {
     type: 'email',
     to: email,
-    from: 'Michael@FreeCodeCamp.com',
+    from: 'Michael@freeCodeCamp.com',
```
Comment (raisedadead) [inline server/boot/certificate.js]: Revert this, and else where.

File: server/boot/certificate.js [hunk 6]
```diff
@@ -238,7 +238,7 @@ export default function certificate(app) {
                   We need your name so we can put it on your certificate. 
                   <a href="https://github.com/settings/profile">Add your
                   name to your GitHub account</a>, then go to your
-                  <a href="https://www.freecodecamp.com/settings">settings
+                  <a href="https://www.freeCodeCamp.com/settings">settings
```
Comment (raisedadead) [inline server/boot/certificate.js]: Revert this, and else where.

**Review by raisedadead — CHANGES_REQUESTED** [review]
1. Revert change in the domain name, email.
2. Exclude the views directory. It has already been changed by a previous PR.

ProTip: `git pull --rebase upstream staging` before pushing to know if something changed while you were working on it.

Thanks.

**Comment by raisedadead** [comment]
@no-stack-dub-sack and while you are at it:

Can you amend you commit to `chore(server): Updated to server to freeCodeCamp` which will help us identify the commit to clearly where it changes things. Sorry for this extra step, but you will notice this will help a lot.

**Comment by no-stack-dub-sack** [comment]
@raisedadead Sounds good, thanks! I will do this this evening

**Comment by raisedadead** [comment]
@no-stack-dub-sack thanks you are the rockstar!

**Comment by no-stack-dub-sack** [comment]
@raisedadead Sorry, but I think I irreparably messed some things up unfortunately... I did a big no-no by accident, and I think I have to start fresh and re-fork the repo. I'm closing this PR - but honestly, not a whole lot lost since the views directory was already done. This will be cleaner once I start over and will do the rest of what's left in the server directory (except for the email domains), then will go on from there...

**Pull request closed without merging**